job-server-extras/test/spark.jobserver/NamedObjectsJobSpec.scala

```diff
@@ -28,10 +28,11 @@ class NamedObjectsJobSpec extends JobSpecBase(JobManagerSpec.getNewSystem) {
 
   val jobName = "spark.jobserver.NamedObjectsTestJob"
 
-  private def getCreateConfig(createDF: Boolean, createRDD: Boolean) : Config = {
+  private def getCreateConfig(createDF: Boolean, createRDD: Boolean, createBroadcast: Boolean) : Config = {
```

Comment:
You could avoid the need to change most of the test by adding a default parameter, i.e. `createBroadcast: Boolean = false`